src/screens/OtherProfile/index.js

```diff
@@ -670,7 +655,7 @@ const OtherProfile = () => {
         <BlockProfile
           onSelect={onBlocking}
           refBlockUser={blockUserRef}
-          username={username}
+          username={params?.data?.username}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The use of `params?.data?.username` instead of the previously set local state `username` is a good change for consistency.


<!-- This is an auto-generated comment by OSS CodeRabbit -->